[LC-748] Add block data and vote data of version 5.

hyerim-kim opened the pull request:
- Add data of version 5 for remove `_` from a variable name of the `round_`.
- Add logic to get a class that is related to voting by version.
- Edit proto file of the gRPC.
- Replace some the `logging` to the `logger`.

hyerim-kim committed:
[LC-748] Add block data and vote data of version 5.

- Add data of version 5 for remove `_` from a variable name of the `round_`.
- Add logic to get a class that is related to voting by version.
- Edit proto file of the gRPC.
- Replace some the `logging` to the `logger`.
- Optimize imports.

hyerim-kim committed:
[LC-748] Edit data type of the `round` from the `int` to the `hex string`.

- Remove unnecessary log.

hyerim-kim committed:
[LC-748] Edit block data to implement `v0.5` by inheritance of `v0.4`.

- Optimized imports in some files.
[LC-748] Edit block data to implement `v0.5` by inheritance of `v0.4`.

- Optimized imports in some files.

hyerim-kim committed:
[LC-748] Edit block data to implement `v0.4` by inheritance of `v0.3`.

- Optimized imports in some files.

hyerim-kim committed:
[LC-748] Replace some code of from using any specific class to using `get_vote_class_by_version`.

windies21 commented on loopchain/baseservice/node_subscriber.py:
```diff
@@ -164,7 +164,8 @@ def __del__(self):
         try:
             votes_serialized = json.loads(votes_dumped)
-            vote = BlockVotes.deserialize_votes(votes_serialized)
+            votes_class = utils.get_vote_class_by_version(block_dict["version"])["BlockVotes"]
```
This file no longer uses 'BlockVotes'. So import section is to be re-organized.

hyerim-kim committed:
[LC-748] Restore the `loopchain.proto` file.

- Optimize imports.

hyerim-kim committed:
[LC-748] Relocate logic of the `get_vote_class_by_version` into the `Vote` and the `Votes` classes.

- Optimize imports.
- Remove the unnecessary log.
- Add some comment for a method.
